### DISCUSSION TITLE
O poder do Browser

### DISCUSSION THREAD

**Discussion by expalmer** [frontendbr/forum#316] [Perguntas e Respostas]
Tudo bem pessoal, gostaria de compartilhar na minha faculdade sobre o poder que temos através das features dos novos navegadores.
Poderiam me das dicas do que eu poderia falar ? Por exemplo, as APIS de audio, vídeo...acelerômetro...WebGL ... Canvas...

**Comment by felipefialho** [comment]
@felquis

**Comment by gabrieldarezzo** [comment]
Um PodCast que eu ouvi recentemente:
http://hipsters.tech/progressive-web-apps-hipsters-03/

@sergiolopes @mauricio

Recomendo.

**Comment by ericdouglas** [comment]
O podcast que o @gabrieldarezzo indicou é bem legal, e [esse vídeo](https://www.youtube.com/watch?v=m2tvYGCdOzs&feature=youtu.be) achei bem interessante também :smile:

**Comment by expalmer** [comment]
Valeu gente, vou dar uma olhada.

**Comment by thiamsantos** [comment]
@expalmer é muito interessante falar de Progressive web apps porque, podemos ver o poder de várias APIs sensacionais trabalhando juntas e ainda mantendo o Progressive Enhancement para que a aplicação não quebre quando o browser não suporta determinados recursos.

Vale a pena dar uma conferida nessa duas palestras sobre o assunto também:
- [Palestra do Sérgio Lopes no Frontinsampa 2015](https://youtu.be/sH7dlRnuh-k) 
- [Palestra do Jake Archibald no Braziljs 2016](https://youtu.be/k6P9ndFNarg)
